src/styles/blocks/card.css

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  justify-content: center;
+}
+
+.card__image {
+  box-sizing: unset;
+  width: 160px;
+  height: 134px;
+  padding: 33px 20px 0;
+}
+
+.card__product-name {
+  padding: 40px 16px 0;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--color-dark-blue);
+}
+
+.card__product-code {
+  padding-top: 4px;
+  padding-left: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
```

Comment:
[font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) 400 is the same as `normal` - default value